README.md

```diff
@@ -14,11 +14,14 @@ Setup
 - Clone repo
 - Change directory into project folder `cd zt-app`
 - `make install`
-- If you do `make start` now, a window will open in your browser. If you click "Run on Andoird" or "Run on iOS", it will tell you how set these up.
+- If you do `make start` now, a window will open in your browser. If you click
+"Run on Andoird" or "Run on iOS", it will tell you how set these up. You can use
+`make start-web` if you want to view in chrome however some styling will be broken.
+
 
 Setup after Hibernation
-- Check correct Node version: 18.3.0 (15.6.22) -> Note: expo-cli has not yet been tested against v18.3.0
-- Update expo-cli - `npm install -g expo-cli`: 5.4.9 (15.6.22)
+- Check correct Node version: 18.4.0 (04.08.22) -> Note: expo-cli has not yet been tested against v18.4.0
```

Comment:
was wondering if it is better to include the `Technical Review (Updates)` part into this one?

Reply:
I'll move it right above but won't squash it into one as we all need to do the setup but only one of us does the technocal review